packages/react/src/components/AssessSolution/AssessSolutionDashboard.tsx

```diff
@@ -84,6 +111,119 @@ const AssessSolutionDashboardInternal: React.FC<Props> = ({ solution, projects }
 
   return (
     <SpaceBetween size="m">
+      <EnterEmailModal
+        visible={emailModal}
+        onCancel={() => setEmailModalVisible(false)}
+        onSaveExit={() => {
+          setEmailModalVisible(false);
+          setFeedbackModalVisible(true);
+        }}
+      ></EnterEmailModal>
+
+      <Modal
+        onDismiss={() => {
+          setFeedbackModalVisible(false);
+          setInputValue("");
+          setCategory("");
+          setIsCategoryEmpty(false);
+          setIsValueEmpty(false);
+        }}
+        visible={feedbackModal}
+        closeAriaLabel="Close modal"
+        size="medium"
+        footer={
+          <Box float="right">
+            <SpaceBetween direction="horizontal" size="xs">
+              <Button
+                variant="link"
+                onClick={() => {
+                  setFeedbackModalVisible(false);
+                  setInputValue("");
+                  setCategory("");
+                  setIsCategoryEmpty(false);
+                  setIsValueEmpty(false);
+                }}
+              >
+                Cancel
+              </Button>
+              <Button
+                variant="primary"
+                onClick={() => {
+                  if (category === "" || category == null) {
+                    setIsCategoryEmpty(true);
+                  }
+                  if (inputValue === "" || inputValue == null) {
+                    setIsValueEmpty(true);
+                  }
+                }}
```

Comment:
Hi Usman, additional logic will be added here to handle input data, right? once you finalize back-end. Just want to confirm.